### ISSUE TITLE
As an ISL user, I want to add a new dog to paw, so I can track their activity with the paw application

### ISSUE THREAD

**Issue by akrawchyk** [istrategylabs/paw#10]
Form fields for add (required*):
- [ ] Dog Name*
- [ ] Owner Name\* (populated by ISL API)
- [ ] Birthday
- [ ] Food restrictions
- [ ] Description
- [ ] Likes
- [ ] Dislikes

**Comment by akrawchyk** [comment]
Adding a few fields here:
- [ ] Veterinarian contact
- [ ] backup owner

**Comment by samyabehary** [comment]
Veterinarian and Backup Owner should both be required fields as well.

**Comment by g3rd** [comment]
Ticket in ISL API: https://github.com/istrategylabs/isl-api/issues/42

**Comment by samyabehary** [comment]
For Veterinarian, is it best to split it into two fields for Vet Name / Office and Phone Number? Or is it alright to keep them all in one line?

**Comment by akrawchyk** [comment]
I believe @g3rd is planning on splitting them like you suggested.

**Comment by samyabehary** [comment]
ok awesome - ill split them in the survey.

**Comment by g3rd** [comment]
Staging has the expanded pet data:
https://isl-api-staging.herokuapp.com/api/pets/

### List JSON:

```
[
    {
        "display_id": null,
        "owner": "taylor",
        "owner_link": "https://isl-api-staging.herokuapp.com/api/v2/employees/taylor/",
        "animal": "dog",
        "name": "Watson",
        "slug": "watson",
        "pet_link": "https://isl-api-staging.herokuapp.com/api/v2/pets/watson/"
    }
]
```

### Detail Pet JSON:

```
{
    "display_id": "39pABg",
    "owner": "taylor",
    "owner_link": "https://isl-api-staging.herokuapp.com/api/v2/employees/taylor/",
    "backup_owner": null,
    "backup_owner_link": null,
    "animal": "dog",
    "name": "Watson",
    "slug": "watson",
    "avatar": "https://isl-api.s3.amazonaws.com:443/pet/watson/tumblr_npn0en9vZ51qgk46eo1_1280_nALwXzu.jpg",
    "avatar_width": 1280,
    "avatar_height": 1206,
    "birthday": null,
    "short_description": "",
    "description": "",
    "food_restrictions": "",
    "likes": "",
    "dislikes": "",
    "veterinarian_name": "",
    "veterinarian_phone_number": "",
    "btle_devices": [
        {
            "device_id": "gallium-po-dedicate-pietism",
            "notes": ""
        }
    ]
}
```

### Detail Employee (Owner) JSON:

Note: This is still going through some changes

```
{
    "display_id": "4vRMxk",
    "user": {
        "username": "chad",
        "email": "chad@isl.co",
        "first_name": "Chad",
        "last_name": "Shryock",
        "is_active": true
    },
    "slug": "chad-shryock",
    "nickname": "",
    "team_display_id": null,
    "team": {
        "group": "Antimatter",
        "slug": "antimatter"
    },
    "job_title": "Creative Technologist",
    "job_title_slug": "creative-technologist",
    "address": "2550 17th St. NW\r\nUnit 215\r\nWashington, DC 20009",
    "phone_number": "(571) 436-2116",
    "personal_email": "gen3rd@gmail.com",
    "conference_phone_number": null,
    "conference_url": null,
    "conference_notes": null,
    "reports_to": "jeremy-carbaugh",
    "avatar": "https://isl-api.s3.amazonaws.com:443/employee/chad/islco-chadShryock2x-500x500.jpg",
    "avatar_width": 500,
    "avatar_height": 500,
    "start_date": "2015-01-26",
    "end_date": null,
    "birthday": "1980-09-26",
    "t_shirt_size": "Small",
    "bio": "Nulla vitae elit libero, a pharetra augue. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Donec ullamcorper nulla non metus auctor fringilla. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Donec sed odio dui.\r\n\r\nCras justo odio, dapibus ac facilisis in, egestas eget quam. Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Vestibulum id ligula porta felis euismod semper. Nullam id dolor id nibh ultricies vehicula ut id elit.",
    "social_networks": [
        {
            "id": 1,
            "social_network": {
                "name": "Facebook",
                "slug": "facebook",
                "base_url": "https://www.facebook.com/"
            },
            "user_name": "chad.shryock"
        },
        {
            "id": 4,
            "social_network": {
                "name": "Instgram",
                "slug": "instgram",
                "base_url": "https://instagram.com/"
            },
            "user_name": "chad"
        },
        {
            "id": 2,
            "social_network": {
                "name": "Twitter",
                "slug": "twitter",
                "base_url": "https://twitter.com/"
            },
            "user_name": "webchad"
        }
    ],
    "is_executive_team": false,
    "is_leadership_council": false
}
```

**Comment by g3rd** [comment]
@akrawchyk how does this look?

**Comment by akrawchyk** [comment]
@g3rd looks like we have everything we need, thanks!

PS - you might want to scrub your emails and info from that example, this is a public repo :stuck_out_tongue:

**Comment by samyabehary** [comment]
Depends on #44
